source/_integrations/music_assistant.markdown

```diff
@@ -183,19 +203,91 @@ Perform a local search on the Music Assistant library.
   - **Description**: When `album` is the `media_type` then this option will restrict the result according to the selection of either album, single, compilation, EP or unknown.
   - **Example**: `album`
 
+#### Example
+
+This example will start playback of ten random tracks.
+
+```yaml
+script:
+  create_random_queue:
+    mode: single
+    sequence:
+      - service: music_assistant.get_library
+        data:
+          limit: 10
+          media_type: track
+          config_entry_id: 01JEXNDHT21V0BHJXM7A5SZANV
+          order_by: random
+        response_variable: random_tracks
+      - repeat:
+          count: "{{ random_tracks['items'] | length }}"
+          sequence:
+            - action: music_assistant.play_media
+              data:
+                media_id: "{{ random_tracks['items'][repeat.index-1].uri }}"
+                media_type: track
+                enqueue: add
+              target:
+                entity_id: media_player.ma_kitchen_speaker
+```
+
 ### Action `music_assistant.get_queue`
 
-Get the queue details of a Music Assistant player queue.
+Get the queue details of a Music Assistant player queue. This provides programmatic access to comprehensive information about the current and next media item in the queue. This information could be used to create a bespoke media dashboard.
 
 - **Data attribute**: `entity_id`
   - **Optional**: No.
   - **Description**: The entity_id of the player holding the queue to be retrieved.
   - **Example**: `media_player.kitchen_speaker`
 
+#### Example
+
+This example sets the name of the currently playing track in an [`input_text`](https://www.home-assistant.io/integrations/input_text/) which could then be used on a dashboard.
+
+```yaml
+script:
+  get_now_playing:
+    mode: queued
+    alias: "Get Now Playing Track Name"
+    sequence:
+      - action: music_assistant.get_queue
+        data:
+          entity_id: media_player.ma_kitchen_speaker
+        response_variable: queue_info
+      - service: input_text.set_value
+        data:
+          entity_id: input_text.now_playing
+          value: "{{ queue_info['media_player.ma_kitchen_speaker'].current_item.name }}"
+```
+  
```

Comment:
I'm not sure if these examples are needed ?
Maybe better to simply make an example of the command and what it outputs ?

Reply:
The quality level item says "The documentation provides automation examples the user can use" so I was trying to find examples that met this. It does also mention using blueprints and linking to those but these seemed small enough and I had a problem with one of them trying to get the HA GUI automation editor to accept the template